src/components/ExpandableInfo.tsx

```diff
@@ -1,216 +1,155 @@
+  Box,
+  Center,
+  ChakraProps,
+  Collapse,
+  forwardRef,
+  Heading,
+  HStack,
+  Icon,
+  Stack,
+  Text,
+  useDisclosure,
+  VStack,
+} from "@chakra-ui/react"
+import { motion } from "framer-motion"
+import { MdExpandMore } from "react-icons/md"
 
-import Icon from "./Icon"
-
-const Card = styled.div<{
-  background: string
-}>`
-  border: 1px solid ${({ theme }) => theme.colors.border};
-  border-radius: 2px;
-  padding: 1.5rem;
-  display: flex;
-  flex-direction: column;
-  margin-bottom: 1rem;
-  background: ${({ background, theme }) =>
-    background ? theme.colors[background] : theme.colors.background};
-  &:hover {
-    img {
-      transform: scale(1.08);
-      transition: transform 0.1s;
-    }
-  }
-`
-
-const Content = styled.div`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  gap: 3rem;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    gap: 2rem;
-    flex-direction: column;
-  }
-`
-
-const TitleContent = styled.div`
-  display: flex;
-  align-items: center;
-  gap: 3rem;
-  width: 100%;
-`
-
-const Title = styled.h2`
-  margin-top: 0rem;
-  margin-bottom: 0.5rem;
-`
-
-const TextPreview = styled.p`
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text200};
-  margin-bottom: 0rem;
-`
-
-const Text = styled(motion.div)`
-  font-size: 16px;
-  font-weight: 400;
-  color: ${(props) => props.theme.colors.text};
-  margin-top: 2rem;
-  border-top: 1px solid ${(props) => props.theme.colors.border};
-  padding-top: 1.5rem;
-`
-
-const Question = styled.div`
-  margin-right: 1rem;
-`
-
-const Header = styled.div`
-  display: flex;
-  width: 100%;
-  margin: 1rem 0;
-  align-items: center;
-  img {
-    margin-right: 1.5rem;
-  }
-`
-
-const ButtonContainer = styled(motion.div)`
-  display: flex;
-  width: 5rem;
-  justify-content: center;
-  align-items: center;
-  min-height: 10rem;
-  cursor: pointer;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    min-height: 100%;
-    height: 100%;
-    width: 100%;
-    margin: 0;
-  }
-  &:hover {
-    svg {
-      transform: scale(1.25);
-      transition: transform 0.1s;
-    }
-  }
-`
-
-export interface IProps {
+export interface IProps extends ChakraProps {
   children?: React.ReactNode
   image?: IGatsbyImageData
   title: ReactNode
   contentPreview: ReactNode
-  background: string
-  forceOpen: boolean
+  background: BackgroundProps["background"]
+  forceOpen?: boolean
   className?: string
 }
 
-const ExpandableInfo: React.FC<IProps> = ({
-  image,
-  title,
-  contentPreview,
-  children,
-  background,
-  forceOpen,
-  className,
-}) => {
-  const [isVisible, setIsVisible] = useState(forceOpen)
-  const expandCollapse = {
-    collapsed: {
-      height: 0,
-      transition: {
-        when: "afterChildren",
-      },
-    },
-    expanded: {
-      height: "100%",
-      transition: {
-        when: "beforeChildren",
-      },
+const ExpandableInfo = forwardRef<IProps, "div">(
```

Comment:
Wrapped component in the Chakra `forwardRef` to allow Chakra props to be used on this component at the parent level.

See [run-a-node page](https://github.com/ethereum/ethereum-org-website/blob/d9f6693d92b66f81fa3dc0416fc26be6d8938156/src/pages/run-a-node.tsx#L539) when migrating the styled component to Chakra.

Reply:
I'm sorry @pettinarip . I made a grave error in my explanation and needed to review `forwardRef`!! 😅 

So yes, the `forwardRef` from Chakra is an enhanced version of React's `forwardRef` (Check out the [source code](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/system/src/forward-ref.tsx))

However, the primary purpose of this version of `forwardRef` is to ensure the `as` prop is passed in at the parent level without type error should the given component need to be a different html element or consume a different component on a particular instance.

For example, if this component normally renders a `div` wrapping the contents but one of the parent components needs it to be an `article` in a given instance.

So doing this:
```tsx
<ExpandableInfo as="article">
```

Will not generate a type error.

See the extended documentation regarding the ["as" prop](https://chakra-ui.com/community/recipes/as-prop)

I now believe this is actually unnecessary here, because there is no negative impact to passing props at the parent level without `forwardRef` so long as the component props are typed correctly, no need for the `as` prop, and no need to pass reference at the parent level.

Now the only other thing related to this discussion is my familiarity with the argument on [discouraging the use of `React.FC`](https://github.com/facebook/create-react-app/pull/8177) and would like your opinion on this! 😄

Reply:
Cool, thanks for the explanation & resource links again.

When we did the TS migration, I didn't know about that discussion about `React.FC`. Later, I saw that CRA was removing the usage of it and it grab my attention. I'm not super deep into the details but I can say that I don't see too much value on using it tbh and totally agree that the implicit `children` prop is not desired.

This is something we could discuss in a GH issue if you have a strong opinion about it and you feel that we should change it 👍🏼

Reply:
Thanks @pettinarip! I do not have a strong preference either way.

In fact current version of the FC does not type the `children`, so you have to explicitly type it either way!

The discussion is indeed old, and React 18 has helped out with this. The implementation of it for the purposes of these components has no impact either way, whether you do this:
```tsx
const someComponent: React.FC<Type> = (props) =>
```
or this:
```tsx
const someComponent = (props: Type) =>
```

But you would still have trouble using generic typing with it though, if any of the components were to use it.

I'm not aware if any of the components actually need that! 😄 

I will revert back to `React.FC` from `forwardRef` as it is not needed here and the component will still accept incoming Chakra Props.